Sources/MongoSwift/MongoCollection+Read.swift

```diff
@@ -332,8 +340,15 @@ public struct FindOptions: Encodable {
     /// A ReadConcern to use for this operation.
     public let readConcern: ReadConcern?
 
+    // swiftlint:disable redundant_optional_initialization
+
     /// A ReadPreference to use for this operation.
-    public let readPreference: ReadPreference?
+    public var readPreference: ReadPreference? = nil
+
+    /// Indicates the type of cursor to use. This value includes both the tailable and awaitData options.
+    public var cursorType: CursorType? = nil
```

Comment:
similar to `ReadPreference` here, is it just easier to not make these decodable, or would it not be useful?

Reply:
So cursor type is just used as a more ergonomic way to set `tailable` and `awaitData` for users. I think the spec tests would directly specify those options in the tests rather than a cursor type, so we don't need to decode it.